WebHostLib/tracker.py

```diff
@@ -228,22 +233,26 @@ def render_timedelta(delta: datetime.timedelta):
 
 
 @pass_context
-def get_location_name(context: runtime.Context, loc: int) -> str:
-    # once all rooms embed data package, the chain lookup can be dropped
+def get_location_name(context: runtime.Context, location_id: int) -> str:
```

Comment:
We've embedded datapackage for 2 versions now, I feel like we can drop these two functions

Reply:
These have been removed.

Reply:
It would be nice if we either disable loading of old seeds, or upgrade seeds to embed the datapackage.

Reply:
Planned.